### DISCUSSION TITLE
fieldGroups Question

### DISCUSSION THREAD

**Discussion by purchasingtoolpak** [flyntwp/flynt#423] [Q&A] [answered]
In the 'inc' folder there is a list of components for pages and another for posts. If I make a new custom post type = 'Movies' - how would I go about adding a new field group file? For example: movieComponents.php .

I tried to substitute movieComponents for postComponents is the postComponents.php file and had no luck. Is there another step I am missing?

Thanks in Advance.

Dave

**Comment by thomasnavarro** [comment, marked as answer]
Hi 👋 

Yes, that's the right way to do it, for each CPT created, you can duplicate the pageComponents.php file and adapt it to your CPT. (don't forget to modify the location field too)

Thomas
![example](https://user-images.githubusercontent.com/36193187/156324568-8246e5fb-0669-47e4-b826-1ee389a24495.png)

**Reply by purchasingtoolpak** [reply]
Forgot to update location field. That worked great.
